packages/health/lib/src/health_factory.dart

```diff
@@ -291,39 +286,28 @@ class HealthFactory {
       'startTimeSec': startTime.millisecondsSinceEpoch ~/ 1000,
       'endTimeSec': endTime.millisecondsSinceEpoch ~/ 1000,
     };
-    final fetchedDataPoints = await _channel.invokeMethod('getData', args);
+    List<Map>? fetchedDataPoints = await _channel.invokeListMethod('getData', args);
```

Comment:
Can we guaranty the return type is List<Map>? ? 
don't you prefer to check to avoid a crush? 

And if we assume we know what we get, why not set it to 
List<Map<String, dynamic>>?

Reply:
We do assume that we get a list of map objects. The reason is that by default maps returning from the native are just `Map<Object?, Object?>` and since Dart 2.0 (I think) implicit downcasting is not allowed. That is why we need to use `map.cast<K, V>()` method.